modules/BrainCRISPR.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+"""This module generates crispr evidence based on Brain Crispr resource."""
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+from urllib.parse import quote
+from requests import post
+import gzip
+import json
+import re
+from functools import reduce
+from typing import List
+
+from pyspark.sql import types as t, functions as f
+from pyspark import SparkFiles
+
+if TYPE_CHECKING:
+    from pyspark.sql import DataFrame, SparkSession
+
+
+class CRISPRBrain:
+    # CRISPR Brain API details. API key is required and also allows for checking for new release.
+    API_SERVER_URL = "https://crisprbrain.org"
+    API_CLIENT_ID = "92f263647c43525d3e4f181aa7e348f26e32129c0f827321d9261cc9765c56c0"
```

Comment:
If they have an API key (vs. just having open access), isn't it supposed to be a secret? Is it OK to have it hardcoded here in the open?

Reply:
This key is inside the [API wrapper code](https://pypi.org/project/crisprbrain/). Not sure if we can or cannot include in the code.... Not sure how to work around it.